packages/project-access/src/constants.ts

```diff
@@ -15,6 +15,27 @@ export const FileName = {
     UI5DeployYaml: 'ui5-deploy.yaml'
 } as const;
 
-export const DirName = {
-    Changes: 'changes'
-} as const;
+export const enum DirName {
+    Changes = 'changes',
+    Sapux = 'src',
```

Comment:
```suggestion
```
No longer used. Should not be included here

Reply:
Adressed in [44d26ce](https://github.com/SAP/open-ux-tools/pull/1801/commits/44d26ce5a994fd469437d09d124d7141afe15ef6). Thanks